packages/esbuild/src/executors/esbuild/lib/build-esbuild-options.ts

```diff
@@ -197,6 +197,9 @@ export function getRegisterFileContent(
   mainFile: string,
   outExtension = '.js'
 ) {
+  // Fix wrong path on Windows host
+  mainFile = mainFile.replaceAll('\\', '/');
+  
```

Comment:
```suggestion
  mainFile = normalizePath(mainFile)
  
```
 We have a helper in @nrwl/devkit for this

Reply:
Done